packages/flutter_tools/lib/src/template.dart

```diff
@@ -26,13 +27,35 @@ import 'dart/package_map.dart';
 /// but without template expansion (data files, etc.), the '.copy.tmpl'
 /// extension may be used.
 ///
+/// Furthermore, templates may contain test files and golden images that are only
```

Comment:
Sorry I haven't been following the discussion that closely - is the implication here that we're going to golden test the list app? If so we should be using skia gold and not the template images, so that we can handle rolling minor breaking changes from the engine

Reply:
> is the implication here that we're going to golden test the list app

Yes, I was asked to add Golden Tests for the list app. Thanks for the heads up, I'll try to figure out how to use Skia Gold instead.